README.md

```diff
@@ -2,64 +2,56 @@
 
 You need node and docker installed to run the chains.
 
-Mosaic will automatically identify if you want to run a geth node or a parity node based on the chain id.
+Mosaic will automatically identify if you want to run a geth node or a parity node based on the chain id or name.
 Any string supported by parity as a network option will start a parity node container.
-Any other string it tries to match to one of the available IDs in the `./utility_chains` directory, e.g. `1406`.
-
-The default ports published with docker on the host are starting from:
-
-* port: `30303`
-* RPC: `8545`
-* Websocket: `8646`
-
-When you start more than one chain, mosaic will increase all port numbers by 1 for each subsequent chain.
-If you already have containers or other services running at the default host ports for publishing, use the `-port`, `-rpc-port`, and/or `-ws-port` options to start at a different port number.
+Any other string it tries to match to one of the available IDs in the `./utility_chains` directory.
+For example, it would match `1406` to `./utility_chains/utility_chain_1406`.
 
 Stopping a container that was started with mosaic completely removes that container from the host.
 Only the content in the mounted data directory remains.
 If you want to keep the container around, for example to debug in the logs after it was stopped automatically after starting, use the `--keep` option of `./mosaic start`.
 
-⚠️ Nodes started with `mosaic` open *all* available APIs. If necessary, make sure your machine is secured.
+⚠️  Nodes started with `mosaic` open *all* available APIs. If necessary, make sure your machine is secured.
 
 ## Running mosaic
 
-Clone `git clone git@github.com:OpenST/mosaic-chains.git` and install `npm install`.
-Run `./mosaic` to get the help output.
+Clone `git clone git@github.com:OpenST/mosaic-chains.git` and install `npm ci`.
+Run `./mosaic --help` to get the help output.
+You can use the `--help` option with any sub-command to get the relevant help for that sub-command, e.g. `./mosaic start --help`.
 
-The default directory for mosaic to store chain data is `~/.mosaic`.
+The default directory for mosaic to store mosaic chain data is `~/.mosaic`.
 You can specify a different directory with the `--mosaic-dir` option.
 
-Examples:
```

Comment:
I think this example on the readme was helpful. When I came across this repo first time, it was simple for me to understand.
Maybe we should retain examples? 

The start example can also mention about  `-port`, `-rpc-port`, and/or `-ws-port` options

Reply:
I removed the examples, because they were the cause of this issue and will be again in the future. Adding the examples in the readme does not give additional information to what I get from the help output of the command. By adding examples, I have redundant information that humans need to keep in sync. That will never happen.

I can add examples again, but we need to understand that they will be outdated and wrong in the future.

Reply:
What do you think @deepesh-kn ?

Reply:
👍